src/modules/extendedCallWindow/docs/fr_FR.md

```diff
@@ -0,0 +1,96 @@
+## Paramètres généraux
+
+### Heure de génération de mission
+Affiche la date et heure de génération de la mission à côté de l'adresse.
+Les deux paramètres suivants permettent d'activer :
+
+* une bordure jaune, si la mission a été générée il y a plus d'un nombre d'heures définit (0 par défaut => pas de bordure).
+* une bordure rouge, si la mission expire au cours de la nuit prochaine.
+
+![Heure génération](bordureJaune.png)
+
+### Vue améliorée des véhicules requis
+Affiche les véhicules requis sous forme de tableau à la place de d'une simple liste.
+Ce tableau prend en compte les véhicules en route, et affiche en vert les besoins satisfaits compte-tenu des véhicules en route et sélectionnés.
+
+![Véhicules requis](vehiculesManquants.png)
+
+### Aperçu des victimes
+Affiche dans l'en-tête de mission les véhicules de secours manquants entre ambulances, équipe médicales et hélicoptères de secours.
+
+![Besoins Victimes](apercuVictimes.png)
+
+### Temps de trajet le plus long
+Affiche, à côté du bouton « Déployer », le temps de trajet le plus long de l'ensemble des véhicules sélectionnés.
```

Comment:
Do you want to stick to the `« Déployer »` Method or use `` `Déployer` `` as we use it?
Our method produces such an output:
![SmartSelect_20210829-152809_Firefox Nightly.jpg](https://user-images.githubusercontent.com/27238105/131252023-01033c46-f56f-412d-b928-7de08c7d4fb5.jpg)